app/site/metrics.js

```diff
@@ -140,6 +140,36 @@ export const weeklyMetrics = {
     format: Formatter.numberWithCommas,
     fullFormat: Formatter.numberWithCommas,
   },
+  belonging_rate_all: {
+    name: "Rate of Belonging of the City",
+    units: "of people",
```

Comment:
This is good, let's keep it like this.

But just something to watch out for, this might confuse visitors:

<img width="1189" alt="Screen Shot 2021-07-16 at 2 40 23 PM" src="https://user-images.githubusercontent.com/11896652/126000474-20d770d9-0c47-4a07-a124-47b1876e2475.png">

The correct interpretation is 52% of Black people in Chicago, but maybe some people might be confused about whether it means 52% of all people in Chicago.

I like the way you have done it, where the population segment is in the metric name and the units is just "of people" so let's keep it that way.

Reply:
Yeah I realized there was no easy way of maintaining consistency with the other Belonging metrics, so I just went ahead and used the titles used by Chicago Health Atlas for the metric name, and stuck with using the "of people" unit from before. If anything we can change them to match the respective metric name.